Release notes

LucasFares opened the pull request:
Updates the plans page to give information on releases. Also updates active development section and makes slight changes to other pages to reflect this change.

LucasFares committed:
Expaded plans page

Added release notes and updated active development notes. Made slight changes on other pages to reflect this change.

LucasFares committed:
Combined with plans page

huss commented on plans.html:
```diff
@@ -1,38 +1,45 @@
+    <li>Added ability to compare a meter's energy usage over a one week interval.</li>
+    <li>Added ability to create, edit, and delete groups of meters.</li>
+    <li>Can now graph a group as a line or bar graph.</li>
+    <li>Can now compare a group's energy usage over a one week interval.</li>
```
The second item refers to comparing a meter. This one refers to a group.  I think we should try to group items to make it easier to understand and to bring out the generality of the system. So, I might say:
Added ability to compare one or more meters and/or groups from current week to previous week.
I also think we should be consistent between "added" and "can now" wording.
I think we should combine the graphing one to so it is all one for meters/groups.

huss commented on plans.html:
```diff
@@ -1,38 +1,45 @@
+
+  <h4>Version 0.2</h4>
+  <ul>
+    <li>Added ability to graph a meter as a line or bar graph.</li>
```
I think this should say 0.2 was the first major release that includes the ability to automatically acquire, store in a database and display electric meter data.

LucasFares committed:
Edited wording

Edited wording of feature list to be more clear and concise.

LucasFares committed:
Removes baselining from features

Removes baselining from features as not yet implemented. The blurb has been saved and can be readded when the feature is ready.
